renovate.json

```diff
@@ -10,7 +10,7 @@
   ],
   "packageRules": [
     {
-      "matchPackageNames": ["jbr", "ktfmt", "golangci-lint"],
+      "matchPackageNames": ["jbr", "ktfmt", "golangci-lint", "eslint"],
```

Comment:
I don't think this is what you want - this disables the "eslint" _Hermit_ package. You probably want the npm package?